packages/graph/lib/graph.ts

```diff
@@ -7,6 +8,8 @@ import { Node } from "./node";
  * Graph represents a
  */
 export class Graph {
+    public nodes_list: string[] = [];
+    public adjacencyList = {};
```

Comment:
Same for adjacency list.  The `Node` type includes a `Channels` Map that contains all of the `Channels` the Node is a member of.